src/AppCache.php

```diff
@@ -79,12 +80,7 @@ protected function invalidate(Request $request, $catch = false)
         }
 
         $response = new Response();
-        $store = $this->getStore();
-        if ($store instanceof RequestAwarePurger) {
-            $result = $store->purgeByRequest($request);
-        } else {
-            $result = $store->purge($request->getUri());
-        }
+        $result = $this->getStore()->purge($request->getUri());
```

Comment:
Where is tag invalidation done now? Or is it another verb used maybe.
I would expect this to not be able to handle tags as it's just the url.

Should we only pass to parent? Or would that mean we don't check IP (worth checking)

Reply:
I have _moved_ this into FOS provided Listeners.